UE/Application/Ports/UserPort.cpp

```diff
@@ -160,10 +160,22 @@ void UserPort::setupIncomingCallButtons(std::function<void()> acceptButtonCallba
     gui.setRejectCallback(rejectButtonCallback);
 }
 
-void UserPort::showTalking()
+void UserPort::showTalking(std::string& text)
 {
     logger.logInfo("UserPort::showTalking");
-    gui.setCallMode();
+    auto& talking = gui.setCallMode();
+    if(text != ""){
```

Comment:
Maybe use something like `not text.empty()` instead of a comparison to the empty string.